lib/v2/screens/explore_screens/vote_screens/proposals/viewmodels/proposals_list_bloc.dart

```diff
@@ -15,17 +15,17 @@ class ProposalsListBloc extends Bloc<ProposalsListEvent, ProposalsListState> {
   Stream<ProposalsListState> mapEventToState(ProposalsListEvent event) async* {
     if (event is InitialLoadProposals) {
       yield state.copyWith(pageState: PageState.loading);
-      Result result = await GetProposalsUseCase().run(state.currentType);
-      yield ProposalsStateMapper().mapResultToState(state, result);
-    }
-    if (event is OnUserProposalsScroll) {
-      Result result = await GetProposalsUseCase().run(state.currentType);
-      yield ProposalsByScrollStateMapper().mapResultToState(state, result);
-    }
-    if (event is OnUserProposalsRefresh) {
-      yield state.copyWith(pageState: PageState.loading);
-      Result result = await GetProposalsUseCase().run(state.currentType);
-      yield ProposalsStateMapper().mapResultToState(state, result);
+      List<Result> results = await GetProposalsUseCase().run(state.currentType);
+      yield ProposalsStateMapper().mapResultToState(state, results);
     }
+    // if (event is OnUserProposalsScroll) {
+    //    List<Result> results await GetProposalsUseCase().run(state.currentType);
+    //   yield ProposalsByScrollStateMapper().mapResultToState(state, results);
+    // }
+    // if (event is OnUserProposalsRefresh) {
+    //   yield state.copyWith(pageState: PageState.loading);
+    //   List<Result> results await GetProposalsUseCase().run(state.currentType);
+    //   yield ProposalsStateMapper().mapResultToState(state, results);
+    // }
```

Comment:
@n13 I comment on this for the moment since I need how often the voice need values ​​will be updated.

Currently on the screen there are 2 operations, they refresh the scroll if it exceeds the first batch and the Refresh Indicator.

Should I just refresh the voice needs with the refresh indicator?

Reply:
yes you need to refresh whenever we refresh proposals since they change with every vote cast.